components/identifier-generator/front/src/feature/tabs/SelectionTab.tsx

```diff
@@ -14,10 +15,12 @@ type SelectionTabProps = {
 };
 
 type ConditionsWithIdentifier = (Condition & {id: string})[];
+type ConditionId = string;
```

Comment:
```suggestion
type ConditionId = string;
type ConditionsWithIdentifier = (Condition & {id: ConditionId})[];
```
If you introduce ConditionId, use it ! (And you should rename it "ConditionIdentifier" by the way)